cypress.config.js

```diff
@@ -15,9 +22,39 @@ module.exports = defineConfig({
     chromeWebSecurity: false,
     setupNodeEvents(on, config) {
       on('task', {
+        wsConnect() {
+          // Check if there is an existing connection and close it if open
+          if (connection && connection.readyState === WebSocket.OPEN) {
```

Comment:
Can you please shorten it to following for brevity?
```suggestion
          if(connection?.readyState === WebSocket.OPEN) {
```